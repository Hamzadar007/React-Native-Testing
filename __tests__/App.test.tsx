import { render } from "@testing-library/react-native";
import App from "../App";

describe("App", () => {
  it("snapshot test", () => {
    const { toJSON } = render(<App />);
    expect(toJSON()).toMatchSnapshot();
  });
});