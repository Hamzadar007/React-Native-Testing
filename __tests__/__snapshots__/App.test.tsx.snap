// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot test 1`] = `
<View>
  <Text>
    Hi i am snapshot
  </Text>
</View>
`;
